packages/slice-machine/src/features/customTypes/customTypesBuilder/GenerateSliceWithAiModal/GenerateSliceWithAiModal.tsx

```diff
@@ -165,7 +165,13 @@ export function GenerateSliceWithAiModal(props: GenerateSliceWithAiModalProps) {
           </Box>
         ) : (
           <ScrollArea>
-            <Box padding={16} height="100%" gap={16} flexWrap="wrap">
+            <Box
+              display="grid"
+              gridTemplateColumns="1fr 1fr"
+              gap={16}
+              height="100%"
```

Comment:
i'd use here `flexGrow={1}`

Reply:
Turns out we don't need either, so I removed `height` ☑️